packages/suite-desktop/src-electron/libs/update-checker.ts

```diff
@@ -0,0 +1,51 @@
+import fs from 'fs';
+import path from 'path';
+import openpgp from 'openpgp';
+
+import { getReleaseNotes } from '@suite/services/github';
+
+const signingKey = process.env.APP_PUBKEY ?? '';
```

Comment:
What's the purpose of the fallback value?

Also wouldn't it be better to fail early here when the file loads and not when a signature is verified? Something like

```suggestion
const signingKey = process.env.APP_PUBKEY;
if (signingKey === undefined) {
    throw new Error()
}
```

Reply:
It was just for TS to cast it as a string. I agree that failing early would be good, I guess only for production builds.

Reply:
> It was just for TS to cast it as a string

`process.env.ANYTHING` is typed as `string | undefined` which exactly matches the semantics: either there's a string value or there isn't. Let's use that, rather than introducing arbitrary special values.

>  guess only for production builds.

I think the fail early should work identically across all builds. The dev builds should be just provided with a mocked public key.

Let's keep build-type conditionals as rare as possible in our codebase to prevent reproducibility issues.

Reply:
Sorry I had something totally different in my mind when I read your reply. You are right, updated: 96c870e